src/ripples/touch-ripple.jsx

```diff
@@ -115,6 +133,46 @@ const TouchRipple = React.createClass({
     this.end();
   },
 
+  //Check if the user seems to be scrolling and abort the animation if so
+  _handleTouchMove(event) {
+    //Stop trying to abort if we're already 300ms into the animation
+    const timeSinceStart = Math.abs(Date.now() - this._startTime);
+    if (timeSinceStart < 300) {
+      this._stopListeningForScrollAbort();
+    }
+
+    //If the user is scrolling...
+    const deltaY = Math.abs(event.touches[0].clientY - this._firstTouchY);
```

Comment:
I believe that it would also be useful to make sure the user is not scrolling along `X`.
For instance, we the `<Tabs />` component, we want to add an horizontal scrolling.